lib/taurus/qt/qtgui/panel/test/test_taurusvalue.py

```diff
@@ -76,6 +77,20 @@ def texts(self, model=None, expected=None, fgRole=None, maxdepr=0):
         self.assertEqual(got, expected, msg)
         self.assertMaxDeprecations(maxdepr)
 
+    def test_labelCaseSensitivity(self):
+        '''Verify that case is respected of in the label widget'''
```

Comment:
PEP8: Better use double-quotes for docstrings (even if in the other methods we are wrongly using single-quotes)

Reply:
Maybe a PEP8 commit fixing all the quotes in this file would be great